services/src/main/java/org/exoplatform/news/NewsService.java

```diff
@@ -25,6 +25,30 @@
    */
   News getNewsById(String newsId, boolean editMode);
 
+  /**
+   * Get the news draft visibility
+   *
+   * @param newsId {@link News} identifier
+   * @return {@link String} The news draft visibility
+   */
+  String getNewsDraftVisibility(String newsId) throws Exception;
```

Comment:
not needed IMHO since you can retrieve this information from news object by calling getNews rest service